### PR TITLE
More debuggers

### PR DESCRIPTION
**Pull request by jrial**
Added two more debuggers:

* PuDB: [fullscreen console-based debugger](https://pypi.python.org/pypi/pudb)
* rPDB: [graphical remote debugger](https://pypi.python.org/pypi/rpdb/)

### CODE REVIEW

**Comment by jrial** [comment]
Note that these snippets don't put extra newlines after the inserted text, to maintain consistency with the rest of the snippets. However, this behaviour was not followed by the original pdb and ipdb snippets, so a separate PR #125 was submitted to fix that.

**Comment by 50Wliu** [comment]
Hi @jrial, can you please rebase this on master now that #125 was merged?  Thanks!

**Comment by jrial** [comment]
@50Wliu Done, do I close this PR and create a new one from master now?

Not entirely sure of the proper PR workflow on GH nor why it's a problem that this is on a topic branch. Feel free to enlighten me.

**Comment by 50Wliu** [comment]
Hmm, I asked because the newline changes are still showing up here even though they were already included in master.  Nothing else is wrong with the PR itself or your workflow :).

File: snippets/language-python.cson [hunk 1]
```diff
@@ -88,10 +88,16 @@
     'body': '{${1:key}: ${2:value} for ${3:key}, ${4:value} in ${5:variable}}'
   'PDB set trace':
     'prefix': 'pdb'
-    'body': 'import pdb; pdb.set_trace()\n'
+    'body': 'import pdb; pdb.set_trace()'
   'iPDB set trace':
     'prefix': 'ipdb'
-    'body': 'import ipdb; ipdb.set_trace()\n'
+    'body': 'import ipdb; ipdb.set_trace()'
+  'rPDB set trace - ':
```
Comment (50Wliu) [inline snippets/language-python.cson]: Why is there a dash ` - ` following this snippet and the PuDB one?

Comment (jrial) [inline snippets/language-python.cson]: Hold on on that merge, seems I committed mid-work. I'll commit the rest as soon as I get home.

**Comment by jrial** [comment]
Trailing dashes removed. I wanted to include a little sentence describing the difference between various debuggers, but decided against it. It would make the text too long, and people can use Google if necessary.

**Comment by 50Wliu** [comment]
Thanks!

**Pull request merged**